plugins/parsers/avro/schema_registry.go

```diff
@@ -0,0 +1,61 @@
+package avro
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/linkedin/goavro/v2"
+	"net/http"
```

Comment:
Please group imports
```suggestion
	"encoding/json"
	"fmt"
	"net/http"

	"github.com/linkedin/goavro/v2"
```